### PR TITLE
Added brush button active class and toggles

### LINKED ISSUE #187: Brush toolbox item should reflect brush state
Steps to reproduce:
1. Launch single class demo
2. Select "Polygon" mode
3. Select "Brush" button in "Brush Tool"

Observe that the "Brush" button doesn't change to indicate it's active. Same with the "Erase" option. A physical change in the button would indicate that the way to de-activate it is to press the button again. My first usage of the brush tool, I attempted to click the "Polygon" mode to de-activate the brush, which didn't do anything. 

tl;dr
- Brush should have visible indication of being active
- Clicking back on the annotation mode should de-activate the brush

### PR DESCRIPTION
**Pull request by joshua-dean**
# Added brush button active class and toggles
## Description
Adds a `brush-button-active` CSS to the "brush" and "erase" buttons. 
It is visually identical to the `locked` state that the "vanish" button uses. 
Closes #187.

## PR Checklist

- [x] Merged latest main
- [x] Version number in `package.json` has been bumped since last release
- [x] Version numbers match between package `package.json` and `src/version.js`
- [x] Ran `npm install` and `npm run build` AFTER bumping the version number
- [x] Updated documentation if necessary (currently just in `api_spec.md`)
- [x] Added changes to `changelog.md`

## Breaking API Changes
no

### CODE REVIEW

**Review by TrevorBurgoyne — COMMENTED** [review]
very cool. one weird thing i noticed is that if you click the brush buttons and/or annotation size buttons and then use a keybind for the same to toggle, then your previously clicked option stays with a black border. not a huge deal, just unexpected and maybe slightly confusing. clearly not introduced in this pr so dont have to fix it now

steps to reproduce:
1) click on the brush/erase button in the toolbox
2) use a keybind (`g` or `e`) to toggle or switch brush mode
3) the `active` class disappears, but a black border remains

brush

**Comment by joshua-dean** [comment]
> very cool. one weird thing i noticed is that if you click the brush buttons and/or annotation size buttons and then use a keybind for the same to toggle, then your previously clicked option stays with a black border. not a huge deal, just unexpected and maybe slightly confusing. clearly not introduced in this pr so dont have to fix it now

Can't reproduce in firefox, but can in chrome :upside_down_face: 

While testing this, I did notice that the toolbox item with the button now appears on load, even though it shouldn't be present for that subtask. I must've done something wrong with the subtask getters. I'll figure that out and re-request review

**Comment by joshua-dean** [comment]
> While testing this, I did notice that the toolbox item with the button now appears on load, even though it shouldn't be present for that subtask. I must've done something wrong with the subtask getters. I'll figure that out and re-request review

So the issue doesn't appear to show up in the current `main`, and doesn't show up in the this branch until the merge commit d53429e. It was a clean merge, so I suppose I'll just have to inspect the entire diff for anything that might have clashed.

**Comment by joshua-dean** [comment]
> So the issue doesn't appear to show up in the current `main`, and doesn't show up in the this branch until the merge commit [d53429e](https://github.com/SenteraLLC/ulabel/commit/d53429e3209d1494433d3369da3a5b8d32c1bd6b). It was a clean merge, so I suppose I'll just have to inspect the entire diff for anything that might have clashed.

Problematic change was [here](https://github.com/SenteraLLC/ulabel/commit/d53429e3209d1494433d3369da3a5b8d32c1bd6b#diff-e2025b1b7bba4202821bfba0846ff59c1901c82c477dcd758cb648ab25a741e1R638).

So why doesn't this show up in `main`? A quick debug trip show me that `after_init` isn't getting called to make the check! So somehow something I've done on this branch brings back the `after_init` call to reveal the bug...

I will be fully deducing this prior to merging. The immediate issue is solved in 37a2436.

**Comment by joshua-dean** [comment]
> I will be fully deducing this prior to merging. The immediate issue is solved in [37a2436](https://github.com/SenteraLLC/ulabel/commit/37a2436000d79073cce9d21931bb96668b91e0b7).

So I've been learning [git worktree](https://git-scm.com/docs/git-worktree) as I've been working on a few branches in parallel. This entire issue was a result of me having a hierarchical layout with all of the other worktrees as subdirectories of the `main` worktree. Files from all open branches were getting bundled into `dist/`.

My first hint should've been that building a fresh `main` resulted in source control changes... 

But a clean main _does_ have this issue, and digging through how multiple branches get built together would be a futile exercise, so I'm calling this good for now.

tl;dr The underlying issue is the only code issue, and was solved in 37a2436. Everything else was a skill issue.

**Review by TrevorBurgoyne — APPROVED** [review]
LGTM 👍
